index.html

```diff
@@ -54,19 +64,28 @@ <h4>Tonic</h4>
             <button class="button-1" type="button"> See Project</button>
           </form>
     </div>
-    </section>
+  </section>
+  </a>
+  <a href="#">
   <section class="Works-card">
     <img class="Snap" src="images/avail.png" alt="Picture">
+    <img class="Snap-2" src="images/art.png" alt="Picture">
     <div class="Block">
-      <h4>Tonic</h4>
+      <h4 class="T-1">Tonic</h4>
+      <h4 class="T-2">Multi-Post Stories</h4>
```

Comment:
Good job for reversing the card, but you can make it much more beautiful if you follow the exact Figma template. So please make sure to fix the left, and top padding issue.

![image](https://user-images.githubusercontent.com/34779928/199811297-83b45d9e-4cb9-472f-b34c-0d38248a079f.png)